### PR TITLE
[DOC][MINOR] pySpark usage - removed repeated keyword causing confusion

### PR DESCRIPTION
**Pull request by manubatham20**
### What changes were proposed in this pull request?
While explaining pySpark usage, use of repeated synonymous words were causing confusion.
Removed "instead of a JAR" word, to keep it more readable.

### Why are the changes needed?
To keep the docs more readable and easy to understand.

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
No code changes, minor documentation change only. No tests added.

### CODE REVIEW

File: docs/submitting-applications.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ locally on your laptop), it is common to use `cluster` mode to minimize network
 the drivers and the executors. Currently, the standalone mode does not support cluster mode for Python
 applications.
 
-For Python applications, simply pass a `.py` file in the place of `<application-jar>` instead of a JAR,
+For Python applications, simply pass a `.py` file in the place of `<application-jar>`,
```
Comment (maropu) [inline docs/submitting-applications.md:79]: I don't have a strong opinion though, this change looks okay. cc: @HyukjinKwon @srowen

**Comment by HyukjinKwon** [comment]
@manubatham20 can you fill the PR description? What confusion this PR fixes?

**Comment by srowen** [comment]
It's trivial but seems OK to me.

**Comment by srowen** [comment]
Jenkins test this please

**Comment by HyukjinKwon** [comment]
It's fine by me. It would be nicer if we fill the PR description correctly though. @manubatham20 can you update PR description?

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**